contracts-periphery/src/UmbraBatchSend.sol

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.13;
 
-import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "../lib/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
```

Comment:
Why did we change the way we're doing the imports here? Just curious.

Reply:
That's because @ syntax was more for working with hardhat easier. And Matt suggested we don't really need it. https://github.com/ScopeLift/umbra-protocol/pull/334#discussion_r869808108

Reply:
Also relevant: https://github.com/ScopeLift/umbra-protocol/pull/334#discussion_r876403812